spec/mailers/notifier_spec.rb

```diff
@@ -318,6 +318,23 @@
     end
   end
 
+  context "limited comments" do
+    let(:commented_limited_post) {bob.post(:status_message, :text => "### Limited headline \r\n It's **really** sunny outside today, and this is a super long status message!  #notreally", :to => :all, :public => false)}
```

Comment:
Line is too long. [219/120]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing inside {.<br>Space missing inside }.